packages/operation/src/evm/utils.ts

```diff
@@ -6,29 +6,22 @@ export function deriveEndpointId(oisTitle: string, endpointName: string): string
 }
 
 export function deriveExtendedPublicKey(mnemonic: string): string {
-  const wallet = ethers.Wallet.fromMnemonic(mnemonic);
-  const hdNode = ethers.utils.HDNode.fromMnemonic(wallet.mnemonic.phrase);
+  const hdNode = ethers.utils.HDNode.fromMnemonic(mnemonic);
   return hdNode.neuter().extendedKey;
 }
 
-export function deriveAirnodeId(masterWalletAddress: string): string {
-  return ethers.utils.keccak256(ethers.utils.defaultAbiCoder.encode(['address'], [masterWalletAddress]));
+export function deriveWalletFromPath(mnemonic: string, provider: ethers.providers.JsonRpcProvider, path?: string) {
```

Comment:
Shouldn't `path` be required here?

Reply:
My idea here was to call this function without a path when deriving the airnode wallet (will use the default path m/44/60/0/0/0) and call it with a path when deriving sponsor wallets.

Maybe this function should be called deriveWallet instead...

Reply:
Either way works for me